Directory.Packages.props

```diff
@@ -21,6 +21,7 @@
         <PackageVersion Include="Avalonia.ReactiveUI" Version="0.10.17" />
 
         <PackageVersion Include="CefGlue.Avalonia" Version="106.5249.0" />
+        <PackageVersion Include="CefGlue.Common" Version="106.5249.1" />
```

Comment:
this is a dependency of CefGlue.Avalonia ... should be brought by it not referenced directly

Reply:
this was the only way I had to bring the target with the move operation that is on the Common. That target was not being brought up when the reference was a transitive one.

Reply:
done ;) thanks